source/jsonnet/england-wales/household/blocks/who-lives-here/visitor_list_collector.jsonnet

```diff
@@ -49,40 +49,95 @@ local rules = import 'rules.libsonnet';
+              id: 'first-name',
+              label: 'First name',
+              mandatory: true,
+              type: 'TextField',
+            },
+            {
+              id: 'last-name',
+              label: 'Last name',
+              mandatory: true,
+              type: 'TextField',
+              guidance: {
+                show_guidance: 'Why we ask about visitors',
+                hide_guidance: 'Why we ask about visitors',
+                contents: [
+                  {
+                    description: 'This is to ensure that everyone is counted in the census. Add any visitors, even if they have been included on a census questionnaire at another address.',
+                  },
+                ],
+              },
+            },
+          ],
+        },
+        when: [
+          {
+            list: 'visitors',
+            condition: 'equals',
+            value: 0,
+          },
         ],
       },
-      answers: [
-        {
-          id: 'first-name',
-          label: 'First name',
-          mandatory: true,
-          type: 'TextField',
-        },
-        {
-          id: 'last-name',
-          label: 'Last name',
-          mandatory: true,
-          type: 'TextField',
-          guidance: {
-            show_guidance: 'Why do I have to include visitors?',
-            hide_guidance: 'Why do I have to include visitors?',
-            contents: [
-              {
-                description: 'We ask for visitor information to ensure that everyone is counted. This helps to produce accurate population estimates. Add any visitors, even if you think they may have been included on a census form at another address.',
-              },
+      {
+        question: {
+          id: 'visitor-add-question',
+          type: 'General',
+          title: {
+            text: 'What is the name of the {ordinality} visitor staying overnight on Sunday {census_date} at {household_address}?',
+            placeholders: [
+              placeholders.censusDate,
+              placeholders.address,
+              placeholders.getListOrdinalityWithoutDeterminer('visitors'),
             ],
           },
+          answers: [
+            {
+              id: 'first-name',
+              label: 'First name',
+              mandatory: true,
+              type: 'TextField',
+            },
+            {
+              id: 'last-name',
+              label: 'Last name',
+              mandatory: true,
+              type: 'TextField',
+              guidance: {
+                show_guidance: 'Why we ask about visitors',
+                hide_guidance: 'Why we ask about visitors',
+                contents: [
+                  {
+                    description: 'This is to ensure that everyone is counted in the census. Add any visitors, even if they have been included on a census questionnaire at another address.',
+                  },
+                ],
+              },
+            },
+          ],
         },
```

Comment:
This is quite a lot of duplication. We should use a similar approach to that used in `england-wales/household/blocks/who-lives-here/anyone_else.jsonnet` and use `question` and `questionTitle` methods.